app/next-client-app/app/(protected)/scanreports/[id]/tables/[tableId]/fields/[fieldId]/beta/columns.tsx

```diff
@@ -28,24 +28,42 @@ export const columns = (
       return (
         <div className="flex items-center gap-2">
           <a
-              className="font-bold underline underline-offset-2"
-              href={`https://athena.ohdsi.org/search-terms/terms?query=${value}`}
-              target="_blank"
-            >
-              {value}
+            className="font-bold underline underline-offset-2"
+            href={`https://athena.ohdsi.org/search-terms/terms?query=${value}`}
+            target="_blank"
+          >
+            {value}
           </a>
           <CopyButton textToCopy={value} />
+        </div>
+      );
+    }
+  },
+  {
+    id: "AI Suggestions",
+    header: ({ column }) => (
+      <DataTableColumnHeader column={column} title="AI Suggestions" />
+    ),
+    enableHiding: true,
+    enableSorting: false,
+    cell: ({ row }) => {
+      const { value, id } = row.original;
+
+      return (
+        <div className="flex justify-start w-full">
           {enableAIRecommendation === "true" && (
-            <AISuggestionsButton
+            <StoredRecommendationsButton
```

Comment:
I think this is two different things though, as in - we want to preserve the existing AI suggestions functionality, whilst this replaces it.
You should be adding a new button, not replacing it.

Reply:
Perfect! I know it looks odd, but only one of these things is close to production.